website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -110,6 +111,7 @@ optional variables can be found in the [variables section](#orca-variables).
 - orca_default_bucket
 - db_user_password
 - db_host_endpoint
+- rds_security_group_id
```

Comment:
Shouldn't this (rds_security_group_id) be under the **Required Values Retrieved from Cumulus Variables** section?

Reply:
Hard to tell... Seems like if that is the case, then much of the DB stuff should be in Cumulus as well.